master/reference/calicoctl/resources/ippool.md

```diff
@@ -34,7 +34,7 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| cidr     | IP range to use for this pool.  | A valid IPv4 or IPv6 CIDR. | string | |
+| cidr     | IP range to use for this pool.  | A valid IPv4 or IPv6 CIDR. Subnet length must be `/26` or less for IPv4 and `/122` or less for IPv6. Must not overlap with the Link Local range `169.254.0.0/16` or `fe80::/10`. | string | |
```

Comment:
IIRC, I thought this had to be a /25 or less. (I didn't test or check this, just going off my memory and could be totally wrong.)

Reply:
nope, it's /26 or less. /26 is the IPAM block size so that's the smallest possible pool (1 block). I also created a /26 pool to be sure